### ISSUE TITLE
[Other] Midterm-#Q.7,8

### ISSUE THREAD

**Issue by susngan1** [goldshakil/ElectricalProgrammingCourse2020#165]
In previous question, you said if y have input character, i have to make error msg. I make code if y is not integer number, then print error msg. But if I input character, program run abnormal. Should I make my code that if input is character, it print error msg?

**Comment by goldshakil** [comment]
Yes if input is character it should 100% generate error and not continue
